docs/deprecations/index.md

```diff
@@ -22,3 +22,6 @@ Deprecations are subject to change in detail or timeframe. If you need help asse
 
 * The Space level `/useronboarding` API endpoint is being removed in future versions of Octopus. It was used internally to improve the user onboarding experience. We have since reworked the new user experience and removed the old endpoint. There is no replacement for this endpoint. We do not expect that anyone outside our internal teams has used this endpoint. If you believe this could negatively affect you, please get in touch with our [support team](https://octopus.com/support).
 
+## Server Extensbility
```

Comment:
@ankithkonda - Is there a version this is planned to be deprecated from? Looking at the existing titles that are H2 headings in this page, it looks like it's designed to be done by the Server version following a heading convention like this:

```md
## Deprecations for 20xx.X
```

Reply:
I would say 2022.4 is when we deprecated it (as part of auth consolidation) and are looking for customers to reach out to us, with end of 2023 being when we expect to remove the functionality. 

I can put this under 2022.4 title 👍

Reply:
I've just pushed up an update to the title :)

Reply:
Looks good, thanks